File: serverCode/Player.py
# Matt Rinne 6/3/2020
# Player class

class Player:
    def __init__(self, uid, nickname, fullName, address):
        self.uid = uid
        self.nickname = nickname
        self.fullname = fullName
        self.hand = ['none', 'none', 'none', 'none']
        self.address = address
    
    def getUid(self):
        return self.uid
    
    def clearHand(self):
        self.hand = ['none', 'none', 'none', 'none']
    
    def addToHand(self, card):
        self.hand.append(card)

File: upTheRiver/Models/Player.swift
//
//  Player.swift
//  upTheRiver
//
//  Created by Matthew Rinne on 6/1/20.
//  Copyright © 2020 Matthew Rinne. All rights reserved.
//

import Foundation

class Player {
    let uid: String
    let nickname: String
    let fullName: String
    
    init(uid: String, nickname: String, fullName: String) {
        self.uid = uid
        self.nickname = nickname
        self.fullName = fullName
    }
}
